coderd/debug.go

```diff
@@ -269,7 +298,7 @@ func loadDismissedHealthchecks(ctx context.Context, db database.Store, logger sl
 		}
 	}
 	if err != nil && !xerrors.Is(err, sql.ErrNoRows) {
-		logger.Error(ctx, "unable to fetch health settings: %w", err)
+		logger.Error(ctx, "unable to fetch health settings", slog.Error(err))
```

Comment:
review: drive-by: this causes slog to panic

Reply:
out of curiosity: why?